gensim/models/word2vec.py

```diff
@@ -121,13 +121,14 @@
 except ImportError:
     from Queue import Queue, Empty
 
-from numpy import exp, dot, zeros, random, dtype, float32 as REAL,\
-    uint32, seterr, array, uint8, vstack, fromstring, sqrt,\
+from numpy import exp, dot, zeros, random, dtype, float32 as REAL, \
+    uint32, seterr, array, uint8, vstack, fromstring, sqrt, \
     empty, sum as np_sum, ones, logaddexp, log, outer
 
 from scipy.special import expit
 
-from gensim import utils, matutils  # utility fnc for pickling, common scipy operations etc
+from gensim import utils, \
```

Comment:
why?

Reply:
We used 120 characters limit (not 80), do not be afraid of long lines

You can check PEP8 with `tox -e flake8` command (with our config for flake8)